src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/job/execution/summary-table.js

```diff
@@ -28,10 +29,16 @@ define(
             toggleDisplayWarning: function (event) {
                 var stepIndex = event.currentTarget.dataset.stepIndex;
                 var warningIndex = event.currentTarget.dataset.warningIndex;
-                var model = this.getFormData();
-                model.stepExecutions[stepIndex].warnings[warningIndex].expanded =
-                    !model.stepExecutions[stepIndex].warnings[warningIndex].expanded;
-
+                if (this.expandedWarnings[stepIndex]) {
+                    if (this.expandedWarnings[stepIndex].includes(parseInt(warningIndex))) {
+                        const warningIndexToRemove = this.expandedWarnings[stepIndex].indexOf(parseInt(warningIndex));
+                        this.expandedWarnings[stepIndex].splice(warningIndexToRemove, 1);
+                    } else {
+                        this.expandedWarnings[stepIndex].push(parseInt(warningIndex));
+                    }
+                } else {
+                    this.expandedWarnings[stepIndex] = [parseInt(warningIndex)];
+                }
```

Comment:
What do you think about this solution:

```suggestion
                const stepIndex = parseInt(event.currentTarget.dataset.stepIndex);
                const warningIndex = parseInt(event.currentTarget.dataset.warningIndex);
                
                this.expandedWarnings[stepIndex] = this.expandedWarnings[stepIndex] || [];
                
                this.expandedWarnings[stepIndex] = this.expandedWarnings[stepIndex].includes(warningIndex) ?
                    this.expandedWarnings[stepIndex].filter(index => index === warningIndex) :
                    [...this.expandedWarnings[stepIndex], warningIndex];
```

Reply:
Many Thanks for your feedback, Julien !
Your proposal seemed good to me but I tested and the "hide item" doesn't work during the execution of the job.
Another problem: once the execution of the job is finished, when I have 2 open warnings and I want to close one, the other closes. I'll work on your piece of code, to find out where the error is.